pvlib/iotools/epw.py

```diff
@@ -67,9 +67,9 @@ def read_epw(filename, coerce_year=None):
     ===============   ======  =========================================
 
 
-    =============================       ==============================================================================================================================================================
+    =============================       ==================================================================================================================================================================================================================
```

Comment:
If you have the time and interest to edit this table (which was cut/pasted from some document) for line length, it would be much appreciated. If not, that's OK, mark the complaining lines with `# noqa: E501`